docs/reference/modules/scripting/painless-debugging.asciidoc

```diff
@@ -38,23 +38,24 @@ POST /hockey/player/1/_explain
 // we have to override it to get a normal shaped response
 
 Which shows that the class of `doc.first` is
-`org.elasticsearch.index.fielddata.ScriptDocValues$Longs` by responding with:
+`org.elasticsearch.index.fielddata.ScriptDocValues.Longs` by responding with:
```

Comment:
Changed this to match the `painless_class` which is what we document in the reference.